acl/authorizer.go

```diff
@@ -0,0 +1,128 @@
+package acl
+
+type EnforcementDecision int
+
+const (
+	// Allow returned from an Authorizer enforcement method indicates
+	// that a corresponding rule was found and that access should be allowed
+	Allow EnforcementDecision = iota
```

Comment:
Do you want the default zero value of `EnforcementDecision` to be `Allow`?

Reply:
Good point. In theory it shouldn't matter but will swap them just in case.